threadit-web/src/pages/PostThread/PostThread.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+import Select from 'react-select'
```

Comment:
can we stick to chakra-ui components? https://chakra-ui.com/docs/components/select

Reply:
I couldn't find a chakra select that included search functionality so I used a react component instead. I'm removing the select for now anyways so should be good to go.